testproject/testproject/templates/500.html

```diff
@@ -10,7 +10,11 @@
 <h1 class="page-header">{% trans "500 - ERROR" %}</h1>
 
 <div class="error">
-  <p>{% trans "Oh god no the demo has failed. Please rest asured that an open source developer is the logs to resolve this, because it's fun!!" %}</p>
+  <p>{% blocktrans %]
```

Comment:
`{% blocktrans trimmed %}`?

Reply:
Oh nice, didn't know that feature! Cheers! :)